test/runtests.jl

```diff
@@ -5,7 +5,7 @@ using Serialization
 
 import DataStructures: IntSet
 
-@test isempty(detect_ambiguities(Base, Core, DataStructures))
+@test [] == detect_ambiguities(Base, Core, DataStructures)
```

Comment:
I'm curious about this change

Reply:
Now, when you have ambigurities in there,
the `@test` macro prints what they are.

Realyl should have been its own commit (if it wasn't already)

Reply:
Can you open an issue for this against Julia? I expanded the `@test` macro to also show `isapprox` and `isequal` and I think `isempty` is common enough we can show the contents there too.

Reply:
https://github.com/JuliaLang/julia/issues/30651

Reply:
I'll note this has now been addressed and you can use `isempty`